lib/pages/chat/chat_input_row.dart

```diff
@@ -133,21 +133,27 @@ class ChatInputRow extends StatelessWidget {
   ChatInputRowMobile _buildMobileInputRow(BuildContext context) {
     return ChatInputRowMobile(
       inputBar: _buildInputBar(context),
-      showEmojiPicker: controller.showEmojiPicker,
-      inputText: controller.inputText,
+      emojiPickerNotifier: controller.emojiPickerNotifier,
       onEmojiAction: controller.emojiPickerAction,
-      onKeyboardAction: controller.inputFocus.requestFocus,
+      onKeyboardAction: () {
+        controller.emojiPickerNotifier.value =
+            !controller.emojiPickerNotifier.value;
```

Comment:
could you please create extension for ValueNotifier<bool>.toggle();
then 
```suggestion
            controller.emojiPickerNotifier.toggle();
```

Reply:
done